src/assets/styles/pages/_resources.scss

```diff
@@ -0,0 +1,490 @@
+          outline-offset: rem(3);
+
+          p {
+            background-color: var(--fl-buttonBgColor, var(--grey-100)) !important;
+            color: var(--fl-buttonFgColor, var(--indigo-800)) !important;
+          }
+
+          svg {
+            background-color: var(--fl-buttonBgColor, var(--grey-100)) !important;
+            stroke: var(--fl-buttonFgColor, var(--indigo-800)) !important;
+          }
+        }
+
+        &:active {
+          background-color: var(--fl-fgColor, var(--indigo-500));
+
+          p {
+            color: var(--fl-bgColor, #fff);
+          }
+
+          svg {
+            stroke: var(--fl-bgColor, #fff);
+          }
+        }
+      }
+
+      .filter-clear-all {
+        margin: auto 0;
+      }
+    }
+  }
+
+  .resources-filtered-number {
+    font-size: rem(16);
+    font-weight: 400;
+    padding: rem(16) 0 rem(24) 0;
+  }
+
+  .card {
+    border-top: 3px solid var(--coral-500);
+    margin-bottom: rem(48);
+
+    .card-image {
+      margin: rem(16);
+      object-fit: contain;
+    }
+
+    .card-title {
+      margin-top: rem(16);
+    }
+
+    .card-tags {
+      display: flex;
+      flex-wrap: wrap;
+      gap: rem(16);
+      margin-bottom: rem(8);
+      margin-top: rem(8);
+
+      .card-tag {
+        background-color: #f2f2f2;
+        border-radius: rem(48);
+        column-gap: rem(4);
+        display: flex;
+        justify-content: center;
+        padding: rem(8);
+        white-space: nowrap;
+
+        p {
+          font-weight: 500;
+          margin-top: 0;
+        }
+
+        svg {
+          height: rem(16);
+          margin: auto;
+          stroke: var(--fl-fgColor, var(--black));
+          width: rem(16);
+        }
+      }
+    }
+
+    .card-publishedYear {
+      font-size: rem(14);
+      margin-top: rem(16);
+    }
+
+    .card-link {
+      margin-top: rem(16);
+
+      a {
+        display: grid;
+        font-weight: 500;
+        gap: rem(4);
+        grid-template-columns: max-content min-content;
+        width: min-content;
+      }
+
+      svg {
+        height: rem(16);
+        margin: auto;
+        width: rem(16);
+      }
+    }
+  }
+}
+
+// Custom styles when "enhance inputs" is enabled
+// Custom checkbox
+.fl-input-enhanced .filter-checkbox + label::before {
+  outline: rem(3) double var(--fl-fgColor, var(--indigo-500));
+}
+
+// Box checked
+.filter .filter-checkbox:checked + label::before {
+  background: var(--fl-fgColor, var(--indigo-500));
+}
+
+// Box active
+.filter .filter-checkbox:active + label::before {
+  background-color: var(--fl-bgColor, var(--indigo-500));
+  border: rem(2.5) solid var(--fl-fgColor, var(--indigo-500));
+}
+
+// Box focus
+.filter .filter-checkbox:focus + label::before,
+.filter .filter-checkbox:hover + label::before {
+  outline: rem(3.5) double var(--fl-fgColor, var(--indigo-500));
+}
+
+// Box focus when "enhance inputs" is enabled
+.fl-input-enhanced .filter-checkbox:focus + label::before,
```

Comment:
My understanding of the design is when enhance inputs is turned on, all checkbox should have a thicker border, not when enhance inputs is on AND focus/hover.

Reply:
Thanks @cindyli, I realized the rules down here were not applied because rules declared in upper part of the file has stronger rules, so I fixed the issue.